lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -73,32 +72,6 @@ void testMain() {
       );
     });
 
-    test('responds correctly to flutter/platform Clipboard.getData failure',
```

Comment:
The reason I deleted it is because it doesn't actually do what it says it does. The setting of the clipboard actually just silently fails in dart2js, so the test never actually did what it said it would do. In dart2wasm, the runtime is in strict mode, so it threw when we tried to set the clipboard. On deeper analysis on dart2js, the whole test is basically bogus, so I just deleted it.